src/Features/Core/Portable/InlineHints/InlineHintHelpers.cs

```diff
@@ -60,5 +60,13 @@ private static async Task<ImmutableArray<TaggedText>> GetDescriptionAsync(Docume
 
             return default;
         }
+
+        public static Func<string>? GetReplacementTextFunction(string parameterName)
+            => () => GetReplacementText(parameterName);
+
+        private static string GetReplacementText(string parameterName)
+        {
+            return $"{parameterName}: ";
```

Comment:
this feels like overkill.  we already have this computed at the side where the replacement function gets created.

Reply:
removed